### DISCUSSION TITLE
Is there support for Advanced Optimus inside G-Helper?

### DISCUSSION THREAD

**Discussion by ethanic17** [seerge/g-helper#2899] [General]
Is there support for Advanced Optimus in G-Helper? I know Ultimate Mode is for turning on the MUX Switch which requires a restart and Standard Mode is Optimus, but is there a way for Ultimate Mode without the restart (e.g. Advanced Optimus)? I know I can use the NVIDIA laptop display mode controller in the system tray but was just wondering if it was also incorporated into G-Helper and if I somehow missed it or have it disabled.

**Comment by seerge** [comment]
@ethanic17 hello, Advanced Optimus is a setting in Nvidia Control Panel (Display Mode). It works absolutely independently from AC or G-Helper when Standard mode is set.

Please make yourself familiar with readme
https://github.com/seerge/g-helper#video_game-gpu-modes
